packages/fxa-auth-server/lib/payments/configuration/base.ts

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import joi from '@hapi/joi';
+
+export type CapabilityConfig = {
+  [key: string]: string;
+};
+
+const capabilitySchema = joi
+  .object()
+  .pattern(joi.string(), joi.string().min(1))
```

Comment:
Shouldn't this pattern be a [regexp where it starts with "capabilities"](https://github.com/mozilla/fxa/blob/f32ae3f1a02969766a7ba4686907f941793a8fa6/packages/fxa-auth-server/lib/routes/validators.js#L429)?

Reply:
Nope, the nice thing about not having a flat namespace in Stripe metadata is that this object itself is only capabilities, so no prefix needed.

Reply:
Ah I see from your test file the expected structure is something like:
```
  capabilities: {
    [oauthClientId | *]: string,
  },
```